packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -421,7 +421,14 @@ const checkOutputDir = (path) => {
     }
 }
 
-const main = (opts) => {
+const main = async (opts) => {
```

Comment:
Note: this method already returns a promise, so changing this to an `async` function is of minimal consequence.